server/test/integration/dashboardTest/submitEditLocationTest.js

```diff
@@ -59,7 +59,6 @@ describe('dashboard.js integration tests: submitEditLocation', () => {
         durationTimer: 90,
         stillnessTimer: 10,
         initialTimer: 9856,
-        alertApiKey: 'newApiKey',
```

Comment:
Funny, I don't think this was even used here originally. I guess it was just a leftover from a copy-and-paste from submitEditClientTest.js.

Reply:
Haha, yeah. While writing this PR, I essentially did a repo-wide search (case-insensitive) for alertapikey and responderpushid, working through every occurrence. It seemed like there was lots of random leftovers :)